testsuite/features/reposync/srv_sync_fake_channels.feature

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2022-2023 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Prepare fake package channels
```

Comment:
Actually, packages and patches.

Reply:
Correct. I will add "patches".

While we are at it, we have an organization problem here:

We have other "fake" channels, for Debian-Like and Red Hat-like. But they are not covered by this file.

I'm not exactly sure why it is so.

Reply:
Oh no, absolutely right...
They are here... reposync/srv_create_repository.feature
(Mine would be the neverend PR, but I will try to fix it on mine)

Reply:
Please, no :smiley:  LOL... just keep it in mind, otherwise you will never end and me neither :smile: